workspace/README.md

```diff
@@ -10,14 +10,18 @@ These are proposals, which may eventually evolve into guidelines, or not.  Eithe
 
 In this directory, name your files
 
-  ``YYYY-MM-something-descriptive``
+  ``YYYY/MM/something-descriptive``
```

Comment:
This is purely a suggestion, can revert it again. Wondering if there will be drafts that are longer running? Will there be stuff that doesn't move on from the workspace where it's worth keeping it ordered by date?